openclsim/plot/step_chart.py

```diff
@@ -0,0 +1,23 @@
+"""Get the step chart of the container levels."""
+
+import matplotlib.pyplot as plt
+
+from .log_dataframe import get_log_dataframe
+
+
+def get_step_chart(simulation_objects):
+    """Get the step chart of the container levels."""
+
+    fig = plt.figure()
+    for obj in simulation_objects:
+        df = get_log_dataframe(obj)
+        container_list = obj.container.container_list
+        for container in container_list:
+            container = f" {container}" if container != "default" else ""
```

Comment:
Exactly, this is not so nice. I think we should make the difference between normal and multicontainers explicit. For example, let a normal container log a float under the key `container level` and let a multicontainer log a dict of floats under the key `multicontainer levels`. Then, you can simply check for the existence of a column `multicontainer levels` and proceed from here.